src/includes/set-fingerprint/rpc/apple.mdx

```diff
@@ -0,0 +1,39 @@
+```swift
+import Sentry
```

Comment:
on Android/Java SDKs we're used to add all the necessary imports, because of https://develop.sentry.dev/sdk/philosophy/#assume-novice-developers
so people can copy/paste the code snippet and just works.

would be the case here for  `NSError`, `NSURLErrorFailingURLErrorKey` or are they inferred automatically in Swift/ObjC?

Reply:
In swift they are inferred automatically, not the case for objc.

But I don't think this is necessary, the snippet cannot be copied as it is to be used in an empty file, a little understanding of development is required to use the sample.

Reply:
> a little understanding of development is required to use the sample.

What this means can vary by person. We try to include imports so that the only understanding customers need is that the import stuff goes on the top, and the snippet elsewhere. And hence remove the chance of using copying code then trying to build or run and not being able to. Which happened in the past.

Reply:
Thanks for pointing that out, @marandaneto. `import Sentry` is all you need for Sentry classes. `NSError` and `NSURLErrorFailingURLErrorKey` are part of the Foundation framework, so you don't need to add imports for them. The code sample compiles if you just have `import Sentry. So this is fine.